### PR TITLE
algolia installed and added in settings.py

### PR DESCRIPTION
**Pull request by DataKhukhua**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/29)

### CODE REVIEW

File: covidX/settings.py [hunk 1]
```diff
@@ -177,3 +178,8 @@
     "SCHEMA_OUTPUT": "schema.json",
     "SCHEMA_INDENT": 2,
 }
+
+ALGOLIA = {
+            'APPLICATION_ID': 'YourApplicationID',
+                'API_KEY': 'YourAdminAPIKey'
```
Comment (codecakes) [inline covidX/settings.py:184]: Codacy found an issue: [Wrong hanging indentation (remove 12 spaces).](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6123426)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6123426&bid=20033555)

File: covidX/settings.py [hunk 2]
```diff
@@ -177,3 +178,8 @@
...
+                }
```
Comment (codecakes) [inline covidX/settings.py:185]: Codacy found an issue: [Wrong hanging indentation.](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6123426)

File: covidX/settings.py [hunk 3]
```diff
@@ -177,3 +178,8 @@
     "SCHEMA_OUTPUT": "schema.json",
     "SCHEMA_INDENT": 2,
 }
+
+ALGOLIA = {
+            'APPLICATION_ID': 'YourApplicationID',
```
Comment (codecakes) [inline covidX/settings.py:183]: Codacy found an issue: [Wrong hanging indentation (remove 8 spaces).](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6123426)

Comment (DataKhukhua) [reply covidX/settings.py:183]: i'm new in open source :octocat:

Comment (codecakes) [reply covidX/settings.py:183]: @DataKhukhua please run "Black" to format this file.

**Review by codecakes — COMMENTED** [review]
@DataKhukhua make the changes and open the branch

**Comment by premalrupnur** [comment]
what am I expected to do in this issue??Because this is  a closed a issue right?

**Comment by premalrupnur** [comment]
I guess this pr already integrates algolia, then what i have to do???

**Comment by premalrupnur** [comment]
Hey I ran black to format this file but no changes  are reflected

**Pull request closed without merging**